src/main/java/com/thoughtworks/aceleradora/controladores/PedidoControlador.java

```diff
@@ -51,24 +51,19 @@ public String pedidoCriados(Breadcrumb breadcrumb, Model modelo) {
         return "pedido/pedidos";
     }
 
-    @GetMapping("/{id}")
+    @GetMapping("/{id}/visualizar-pedido")
     public String visualizarPedido(@PathVariable("id") Long id, Model modelo, Breadcrumb breadcrumb) {
 
-        breadcrumb.aproveitar(partesComunsDoBreadCrumb).pagina("Pedidos", "/pedidos").pagina("Visualizar pedido",
+        breadcrumb.aproveitar(partesComunsDoBreadCrumb).pagina("Pedidos", "/pedidos").pagina("Visualizar Pedido",
```

Comment:
Dá pra a gente espaçar aqui da mesma maneira que fazemos nos outros?
```suggestion
        breadcrumb
        .aproveitar(partesComunsDoBreadCrumb)
        .pagina("Pedidos", "/pedidos")
        .pagina("Visualizar Pedido", "/pedidos");
```